https://github.com/Atoptool/atop/pull/289

From 8411f81a202c9c39aebcc637ef94cc3d454cbaaf Mon Sep 17 00:00:00 2001
From: Marc 'Zugschlus' Haber <mh+github@zugschlus.de>
Date: Sat, 6 Jan 2024 07:03:53 +0100
Subject: [PATCH] set LDFLAGS with +=, not overwriting existing variable

this allows the Debian build process to set hardening flags
--- a/Makefile
+++ b/Makefile
@@ -21,7 +21,7 @@ PMPATH2  = /usr/lib64/pm-utils/sleep.d
 PMPATHD  = /usr/lib/systemd/system-sleep
 
 CFLAGS  += -O2 -I. -Wall $(shell pkg-config --cflags glib-2.0)  -Wno-stringop-truncation -Wmissing-prototypes -Wmissing-declarations -Wformat-security # -DNOPERFEVENT   # -DHTTPSTATS
-LDFLAGS  = $(shell pkg-config --libs glib-2.0)
+LDFLAGS += $(shell pkg-config --libs glib-2.0)
 OBJMOD0  = version.o
 OBJMOD1  = various.o  deviate.o   procdbase.o
 OBJMOD2  = acctproc.o photoproc.o photosyst.o  rawlog.o ifprop.o parseable.o

